src/ActionsTests/Contexts/ScopedActionContextTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Axe.Windows.Actions;
+using Axe.Windows.Actions.Contexts;
+using Axe.Windows.Desktop.UIAutomation.CustomObjects;
+using Axe.Windows.Desktop.UIAutomation.TreeWalkers;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Threading;
+
+namespace Axe.Windows.ActionsTests.Contexts
+{
+    [TestClass]
+    public class ScopedActionContextTests
```

Comment:
Might be worth adding test coverage that the cancellation token passed in is the same as `TreeWalkerDataContext.CancellationToken`. Otherwise very thorough and great to see more test coverage being added!

Reply:
Added